test/scripts/streaming/test-streaming-detection.py

```
@@ -0,0 +1,679 @@
+        self.print_color(PURPLE, "=" * 60)
+        print()
+        
+    def check_health(self) -> bool:
+        """Check if Olla is available"""
+        self.print_color(YELLOW, "Checking Olla availability...")
+        try:
+            response = requests.get(f"{self.base_url}/internal/health", timeout=5)
+            if response.status_code == 200:
+                self.print_color(GREEN, "[OK] Olla is reachable")
+                return True
+        except Exception as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)